src/Host/Client/Impl/Definitions/IRBlobService.cs

```diff
@@ -7,8 +7,8 @@
 
 namespace Microsoft.R.Host.Client {
     public interface IRBlobService {
-        Task<long> SendBlobAsync(byte[] data, CancellationToken cancellationToken = default(CancellationToken));
-        Task<IReadOnlyList<IRBlob>> GetBlobAsync(IEnumerable<long> blobIds, CancellationToken cancellationToken = default(CancellationToken));
-        Task DestroyBlobAsync(IEnumerable<long> blobIds, CancellationToken cancellationToken = default(CancellationToken));
+        Task<ulong> CreateBlobAsync(byte[] data, CancellationToken cancellationToken = default(CancellationToken));
+        Task<byte[]> GetBlobAsync(ulong blobId, CancellationToken cancellationToken = default(CancellationToken));
+        Task DestroyBlobAsync(ulong[] blobIds, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
No particular reason, I just happened to fork before the change from array to enumerable happened in master. I'll change it to enumerable.

I'll also rename the method to `DestroyBlobsAsync`, since it's operating on multiple blobs.